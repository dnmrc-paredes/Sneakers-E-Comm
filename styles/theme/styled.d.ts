import 'styled-components'

declare module 'styled-components' {
  export interface DefaultTheme {
    colors: {
      orange: string
      pale_orange: string
      light_gray: string
    }
    fonts: {
      nunito: string
      kumbh: string
    }
  }
}
