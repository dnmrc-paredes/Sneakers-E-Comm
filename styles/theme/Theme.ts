export const Theme = {
  colors: {
    orange: 'hsl(26, 100%, 55%)',
    pale_orange: 'hsl(25, 100%, 94%)',
    light_gray: 'hsl(220, 14%, 75%)'
  },
  fonts: {
    nunito: 'Nunito Sans',
    kumbh: 'Kumbh Sans'
  }
}
