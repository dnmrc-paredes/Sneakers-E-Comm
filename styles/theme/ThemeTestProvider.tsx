import React from 'react'
import { ThemeProvider } from 'styled-components'
import { Theme } from './Theme'

export const ThemeTestProvider = ({
  children
}: {
  children: React.ReactNode
}) => {
  return <ThemeProvider theme={Theme}>{children}</ThemeProvider>
}
