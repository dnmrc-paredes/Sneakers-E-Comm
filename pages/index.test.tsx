import React from 'react'
import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'
import { ThemeTestProvider } from '../styles/theme/ThemeTestProvider'
import Index from './index'

describe('Home page', () => {
  let count = 0
  let cartItems = 0
  const setCartItems = jest.fn(() => cartItems++)
  const setCount = jest.fn(() => count++)

  it('It will render/show the home page', () => {
    const { getByTestId } = render(
      <ThemeTestProvider>
        <Index count={count} setCartItems={setCartItems} setCount={setCount} />
      </ThemeTestProvider>
    )
    expect(getByTestId('add-to-cart')).toBeInTheDocument()
    expect(getByTestId('product-info')).toBeInTheDocument()
  })

  it('It will match snapshot', () => {
    const tree = renderer.create(
      <ThemeTestProvider>
        <Index count={count} setCartItems={setCartItems} setCount={setCount} />
      </ThemeTestProvider>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
