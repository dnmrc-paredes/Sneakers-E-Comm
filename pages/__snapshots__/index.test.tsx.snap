// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page It will match snapshot 1`] = `
<div
  className="Pages__Container-sc-qwt8hg-0 iDTxxF"
>
  <div
    className="Pages__ProductContainer-sc-qwt8hg-1 GpcSr"
  >
    <div
      className="Pages__ImagesContainer-sc-qwt8hg-2 kEsvXl"
    >
      <div
        className="Pages__BigImageContainer-sc-qwt8hg-3 dPRwcM"
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAwIiBoZWlnaHQ9IjQwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="Product 1"
            data-nimg="intrinsic"
            decoding="async"
            onClick={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="Product 1"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              onClick={[Function]}
              src="/_next/image?url=%2Fimage-product-1.jpg&w=828&q=75"
              srcSet="/_next/image?url=%2Fimage-product-1.jpg&w=640&q=75 1x, /_next/image?url=%2Fimage-product-1.jpg&w=828&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <div
        className="Pages__ThumbnailContainer-sc-qwt8hg-4 bIrELa"
      >
        <div
          className="Pages__Thumbnails-sc-qwt8hg-5 duXmfa active"
          onClick={[Function]}
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iOTAiIGhlaWdodD0iOTAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="Product 1"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="Product 1"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fimage-product-1-thumbnail.jpg&w=256&q=75"
                srcSet="/_next/image?url=%2Fimage-product-1-thumbnail.jpg&w=96&q=75 1x, /_next/image?url=%2Fimage-product-1-thumbnail.jpg&w=256&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </div>
        <div
          className="Pages__Thumbnails-sc-qwt8hg-5 duXmfa"
          onClick={[Function]}
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iOTAiIGhlaWdodD0iOTAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="Product 2"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="Product 2"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fimage-product-2-thumbnail.jpg&w=256&q=75"
                srcSet="/_next/image?url=%2Fimage-product-2-thumbnail.jpg&w=96&q=75 1x, /_next/image?url=%2Fimage-product-2-thumbnail.jpg&w=256&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </div>
        <div
          className="Pages__Thumbnails-sc-qwt8hg-5 duXmfa"
          onClick={[Function]}
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iOTAiIGhlaWdodD0iOTAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="Product 3"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="Product 3"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fimage-product-3-thumbnail.jpg&w=256&q=75"
                srcSet="/_next/image?url=%2Fimage-product-3-thumbnail.jpg&w=96&q=75 1x, /_next/image?url=%2Fimage-product-3-thumbnail.jpg&w=256&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </div>
        <div
          className="Pages__Thumbnails-sc-qwt8hg-5 duXmfa"
          onClick={[Function]}
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iOTAiIGhlaWdodD0iOTAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="Product 4"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="Product 4"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fimage-product-4-thumbnail.jpg&w=256&q=75"
                srcSet="/_next/image?url=%2Fimage-product-4-thumbnail.jpg&w=96&q=75 1x, /_next/image?url=%2Fimage-product-4-thumbnail.jpg&w=256&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </div>
      </div>
    </div>
    <div
      className="Pages__InfoContainer-sc-qwt8hg-6 dWWyPi"
      data-testid="product-info"
    >
      <p
        className="Pages__SmallText-sc-qwt8hg-7 hljCWo"
      >
         Sneaker Company 
      </p>
      <h1
        className="Pages__ProductTitle-sc-qwt8hg-8 iliopU"
      >
         Fall Limited Edition Sneakers 
      </h1>
      <p
        className="Pages__ProductInfo-sc-qwt8hg-9 dxMPuo"
      >
        These low-profile sneakers are your perfect casual wear companion. Featuring a durable rubber outer sole, they’ll withstand everything the weather can offer.
      </p>
      <p
        className="Pages__Price-sc-qwt8hg-10 cVqsEK"
      >
        $125.00 
        <span
          className="discount"
        >
           50% 
        </span>
      </p>
      <p
        className="Pages__Discount-sc-qwt8hg-11 gJddHI"
      >
         $250.00 
      </p>
      <div
        className="Pages__ButtonContainer-sc-qwt8hg-12 bKhJpq"
      >
        <div
          className="Pages__Buttons-sc-qwt8hg-13 bZjPAx"
        >
          <button
            className="Pages__Minus-sc-qwt8hg-15 fIZRuM"
            data-testid="minusBtn"
            onClick={[Function]}
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "inline-block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "overflow": "hidden",
                  "padding": 0,
                  "position": "relative",
                  "width": "initial",
                }
              }
            >
              <span
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              >
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTUiIGhlaWdodD0iNSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                  style={
                    Object {
                      "background": "none",
                      "border": 0,
                      "display": "block",
                      "height": "initial",
                      "margin": 0,
                      "maxWidth": "100%",
                      "opacity": 1,
                      "padding": 0,
                      "width": "initial",
                    }
                  }
                />
              </span>
              <img
                alt="Minus"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
              <noscript>
                <img
                  alt="Minus"
                  data-nimg="intrinsic"
                  decoding="async"
                  loading="lazy"
                  src="/_next/image?url=%2Ficon-minus.svg&w=32&q=75"
                  srcSet="/_next/image?url=%2Ficon-minus.svg&w=16&q=75 1x, /_next/image?url=%2Ficon-minus.svg&w=32&q=75 2x"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": undefined,
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
              </noscript>
            </span>
          </button>
          <div
            className="Pages__Count-sc-qwt8hg-14 imGiZA"
          >
            0
          </div>
          <button
            className="Pages__Plus-sc-qwt8hg-16 cirGoH"
            data-testid="plusBtn"
            onClick={[Function]}
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "inline-block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "overflow": "hidden",
                  "padding": 0,
                  "position": "relative",
                  "width": "initial",
                }
              }
            >
              <span
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              >
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTUiIGhlaWdodD0iMTUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                  style={
                    Object {
                      "background": "none",
                      "border": 0,
                      "display": "block",
                      "height": "initial",
                      "margin": 0,
                      "maxWidth": "100%",
                      "opacity": 1,
                      "padding": 0,
                      "width": "initial",
                    }
                  }
                />
              </span>
              <img
                alt="Plus"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
              <noscript>
                <img
                  alt="Plus"
                  data-nimg="intrinsic"
                  decoding="async"
                  loading="lazy"
                  src="/_next/image?url=%2Ficon-plus.svg&w=32&q=75"
                  srcSet="/_next/image?url=%2Ficon-plus.svg&w=16&q=75 1x, /_next/image?url=%2Ficon-plus.svg&w=32&q=75 2x"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": undefined,
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
              </noscript>
            </span>
          </button>
        </div>
        <button
          className="Pages__AddToCart-sc-qwt8hg-17 jCCWjn"
          data-testid="add-to-cart"
          onClick={[Function]}
        >
          Add to cart
        </button>
      </div>
    </div>
  </div>
</div>
`;
